runtime/src/bank.rs

```diff
@@ -106,7 +105,7 @@ impl Bank {
     pub fn new_from_parent(parent: &Arc<Bank>) -> Self {
         let mut bank = Self::default();
         bank.last_id_queue = RwLock::new(parent.last_id_queue.read().unwrap().clone());
-        bank.parent_hash = parent.hash_internal_state();
+        *parent.hash.write().unwrap() = parent.hash_internal_state();
```

Comment:
this is ugly because we might have 2 new_from_parent() calls on the same Bank...  what if we forced folks to make Banks eligible to be parents by first freeze()ing them?

Reply:
How about I just check that it's not Hash::default first?

Reply:
and leave a TODO?